lib/classes/PluginManager.test.js

```diff
@@ -5,7 +5,7 @@ const PluginManager = require('../../lib/classes/PluginManager');
 const Serverless = require('../../lib/Serverless');
 const CLI = require('../../lib/classes/CLI');
 const Create = require('../../lib/plugins/create/create');
-
+const md5 = require('md5');
```

Comment:
Here's what I've used in the past:

```
const crypto = require('crypto');

const hash = crypto.createHash('sha256').update('foo bar').digest('hex');

console.log(hash);
```

This uses `sha256` instead of `md5` since `md5` is broken and has known collisions.